src/shared/lib/oql/oqlfilter.ts

```diff
@@ -261,7 +262,11 @@ export function doesQueryContainMutationOQL(oql_query: string): boolean {
                     break;
                 } else if (alteration.alteration_type === 'any') {
                     // any DRIVER specification, which includes mutation
-                    if (alteration.modifiers.indexOf('DRIVER') > -1) {
+                    if (
+                        alteration.modifiers.findIndex(
+                            m => m.type === 'DRIVER'
+                        ) > -1
```

Comment:
I prefer `find` over `findIndex` if we don't need to use the index anywhere else:
```suggestion
                        alteration.modifiers.find(
                            m => m.type === 'DRIVER'
                        )
```